src/api/channelApi.js

```diff
@@ -0,0 +1,84 @@
+const getChannel = async (projectId) => {
+  const request = {
+    method: "GET",
+    credentials: "include",
+  };
+
+  const response = await fetch(
+    `${process.env.REACT_APP_SERVER_URL}/projects/${projectId}`,
+    request
+  );
+
+  const data = await response.json();
+
+  return data.channels;
+};
+
+const createChannel = async ({ projectId, payload }) => {
+  const request = {
+    method: "POST",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(payload),
+  };
+
+  const response = await fetch(
+    `${process.env.REACT_APP_SERVER_URL}/projects/${projectId}/channels`,
+    request
+  );
+
+  const data = await response.json();
+
+  if (response.status >= 300) {
+    throw new Error(data.message);
+  }
```

Comment:
```suggestion
  if (response.status >= 400) {
    throw new Error(data.message);
  }
```
논의했던대로 `400`번부터  에러 처리 해주면 좋을 것 같습니다!

Reply:
위에 형우님 코멘트에 답변달았습니다